RULES_DESCRIPTIONS.md

```diff
@@ -186,7 +186,16 @@ _Configuration_: N/A
 
 _Description_: By [convention](https://github.com/golang/go/wiki/CodeReviewComments#contexts), `context.Context` should be the first parameter of a function. This rule spots function declarations that do not follow the convention.
 
-_Configuration_: N/A
+_Configuration_:
```

Comment:
the README.md must be also updated to document this new configuration for the rule